src/modules/vendor/pages/ni/management/CoursesDirectory.vue

```diff
@@ -164,17 +164,14 @@ export default {
     },
     isForthcoming (course) {
       const noSlot = !course.slots.length;
-      const noSlotsHappened = !course.slots.some((sameDaySlots) =>
-        this.happened(sameDaySlots));
+      const noSlotsHappened = !course.slots.some((sameDaySlots) => this.happened(sameDaySlots));
 
       return noSlot || noSlotsHappened;
     },
     isInProgress (course) {
       const atLeastOneSlot = course.slots.length;
-      const atLeastOneSlothappened = course.slots.some((sameDaySlots) =>
-        this.happened(sameDaySlots));
-      const notEverySlotsHappened = course.slots.some((sameDaySlots) =>
-        !this.happened(sameDaySlots));
+      const atLeastOneSlothappened = course.slots.some((sameDaySlots) => this.happened(sameDaySlots));
+      const notEverySlotsHappened = course.slots.some((sameDaySlots) => !this.happened(sameDaySlots));
```

Comment:
idem

Reply:
Je crois qu'on peut pas faire ça pour (sameDaySlots) => !this.happened(sameDaySlots) car comme on applique '!' on a !this.happened(sameDaySlots) qui n'est plus une fct.

Reply:
en effet pour le !this.happened ca ne fonctionne pas ><

Reply:
en effet '-_-